git_wrapper/branch.py

```diff
@@ -228,7 +234,7 @@ def abort_patch_apply(self):
         try:
             self.git_repo.git.am('--abort')
         except git.GitCommandError as ex:
-            msg = "Failed to abort git am operation. Error: {0}".format(ex)
+            msg = "Failed to abort git am operation. Error: {ex}"
```

Comment:
Ah, just spotted a missing 'f' here - sorry for missing it the first time around.